migtests/scripts/run-validate-assessment-report.sh

```diff
@@ -60,7 +60,7 @@ main() {
 	yb-voyager version
 
 	step "Assess Migration"
-	if [ "${SOURCE_DB_TYPE}" = "postgresql" ]; then
+	if [ "${SOURCE_DB_TYPE}" = "postgresql" ] || [ "${SOURCE_DB_TYPE}" == "oracle" ]; then
```

Comment:
nit: you can probably remove this condition from this script. This script is about run-validate-assessment-report, and it should be concerned only with that. 

We should take care about pg,oracle but not mysql at the test invocation level. I.e we will only run this test for pg/oracle.

Reply:
+1